composer/loggers/wandb_logger.py

```diff
@@ -36,15 +42,22 @@ class WandBLogger(LoggerDestination):
             stored, which may discard pertinent information. For example, when using
             Deepspeed ZeRO, it would be impossible to restore from checkpoints without
             artifacts from all ranks (default: ``False``).
-        init_params (Dict[str, Any], optional): Parameters to pass into
+        init_kwargs (Dict[str, Any], optional): Any additional init kwargs
             ``wandb.init`` (see
             `WandB documentation <https://docs.wandb.ai/ref/python/init>`_).
     """
 
-    def __init__(self,
-                 log_artifacts: bool = False,
-                 rank_zero_only: bool = True,
-                 init_params: Optional[Dict[str, Any]] = None) -> None:
```

Comment:
isn't it better to bundle all `wandb` init params together, rather than have some that are in the WandBLogger constructor, but others that must be appsed via init_params?

Reply:
The original reasoning for listing out some of the wandb imports was so they could be set via CLI overrides with yahp. YAHP does not support setting dictionaries via CLI args. I could leave it as it was before, or replace these parameters with the dict, or leave it like it is now -- don't have strong preferences here. Thoughts?

Reply:
OK, I will leave this as is!